draft-ietf-httpbis-rfc6265bis.md

```diff
@@ -1820,44 +1820,41 @@ IDNA2003 {{RFC3490}}.
 
 # Privacy Considerations
 
-Cookies are often criticized for letting servers track users. For example, a
-number of "web analytics" companies use cookies to recognize when a user returns
-to a web site or visits another web site. Although cookies are not the only
-mechanism servers can use to track users across HTTP requests, cookies
-facilitate tracking because they are persistent across user agent sessions and
-can be shared between hosts.
+The primary privacy risk associated with Cookies is their ability to correlate user activity. This can happen on a single site, but is most problematic when activity is tracked across different, seemingly unconnected Web sites to build a profile of the user.
+
+Over time, this capability (warned against explicitly in {{?RFC2109}} and all of its successors) has become widely used to authenticate users across sites, to combat fraud and other forms of undesirable traffic, to target advertisements at specific users or at users with specified attributes, to measure how often ads are shown to users, and to recognise when an ad resulted in a change in user behavior.
+
+While in some instances these purposes may not be privacy-invasive, the potential abuses of tracking imply a need for significant limitations on cookies. However, any such limitations need to be imposed with care, because cookies are widely used for a variety of purposes that are critical to the functioning of a Web site (and the Web).
```

Comment:
I think that you are making the wrong distinction.  The purposes are always tracking, but you are implying a value judgment they are objectionable from a privacy standpoint (though this is of course subjective).

```suggestion
Though some of these purposes might not be objectionable from a privacy perspective, the potential for this tracking capability of cookies to be abused is being carefully reviewed. Some client implementations are deploying constraints on cookies with an aim to limit the potential for abuse.  These limitations need to be carefully balanced against the disruption that might result, especially where Web sites depend on preexisting features of cookies for critical functions.
```

Reply:
Went in a different direction - see revision.